src/lib/registration/addon.rb

```diff
@@ -58,6 +59,16 @@ def selected
         @selected ||= []
       end
 
```

Comment:
I would add here a little headline like : returns products which are registered but not installed.